components/appmgt/org.wso2.carbon.appmgt.api/pom.xml

```diff
@@ -24,6 +24,11 @@
     <name>WSO2 Carbon - Api for APP Management</name>
 
     <dependencies>
+        <dependency>
+            <groupId>com.googlecode.json-simple.wso2</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.wso2v1</version>

```

Comment:
dependency versions should not be mentioned in the child poms. They should be defined only in the dependencyManagement section in the top level pom.